packages/react/src/components/buttons/abstract-button.tsx

```diff
@@ -18,10 +40,10 @@ export const defaultButtonStyles = css<{ isMobile: boolean }>`
     justify-content: center;
     letter-spacing: ${({ isMobile }) => (isMobile ? 0.033125 : 0.025)}rem;
     line-height: ${({ isMobile }) => (isMobile ? 1.5 : 1)}rem;
-    min-height: ${({ isMobile }) => (isMobile ? 48 : 32)}px;
+    min-height: ${(props) => getButtonMinHeight(props)}px;
```

Comment:
Aussi quand tu fais juste passer les props à la fonction, tu peux faire ceci
```suggestion
    min-height: ${getButtonMinHeight}px;
```